tritium-lib/src/test/java/com/palantir/tritium/proxy/InstrumentationTest.java

```diff
@@ -575,11 +587,13 @@ void testStackDepth_notFunctionOfHandlers() {
         assertThat(singleHandlerInstrumentedStackTrace).hasSameSizeAs(multipleHandlerInstrumentedStack);
     }
 
-    public interface StackTraceSupplier extends Supplier<StackTraceElement[]> {}
+    public interface StackTraceSupplier {
+        StackTraceElement[] get();
+    }
```

Comment:
Modified to avoid lambda class 'hidden' frames. They both work, the `extends Supplier` version has an extra stack frame.